package.json

```diff
@@ -47,5 +47,6 @@
   "workspaces": [
     "packages/*"
   ],
-  "dependencies": {}
+  "dependencies": {},
+  "version": "0.23.5-alpha.0"
```

Comment:
why is this version here?

Reply:
This is the version I will release when I have the approve. for testing in the QA environment.